test/dynamo/test_cudagraphs.py

```diff
@@ -10,7 +10,16 @@
 import torch._dynamo.test_case
 import torch._dynamo.testing
 from torch._dynamo.testing import same
-from torch.testing._internal.common_utils import skipIfRocm, TEST_WITH_ROCM
+from torch.testing._internal.common_utils import (
+    skipIfRocm,
+    TEST_CUDA_GRAPH,
+    TEST_WITH_ROCM,
+)
+
+if not TEST_CUDA_GRAPH:
+    if __name__ == "__main__":
+        sys.exit(0)
```

Comment:
Same as the one below, please add it to `if __name__ == "__main__"` at the bottom of the file